src/explore-education-statistics-admin/src/pages/publication/components/PublicationManageTeamAccessTab.tsx

```diff
@@ -0,0 +1,54 @@
+import { BasicPublicationDetails } from '@admin/services/publicationService';
+import React from 'react';
+import Details from '@common/components/Details';
+import orderBy from 'lodash/orderBy';
+import ReleaseContributorPermissions from '@admin/pages/publication/components/ReleaseContributorPermissions';
+
+export interface Props {
+  publication: BasicPublicationDetails;
+}
+
+const PublicationManageTeamAccessTab = ({ publication }: Props) => {
+  if (
+    publication.releases === undefined ||
+    (publication.releases && publication.releases.length === 0)
+  ) {
+    return <p>No releases!</p>;
+  }
+
+  const sortedReleases = orderBy(publication.releases, r =>
+    parseInt(r.releaseName, 10),
+  );
+  const newestRelease = sortedReleases.shift();
+
+  if (newestRelease === undefined) {
+    return <p>No releases!</p>;
+  }
+
+  return (
+    <>
+      <h2>Update access for latest release ({newestRelease?.title})</h2>
+      <p>
+        Allow team members to be able to access and edit this release. You can
+        also{' '}
+        <a href="#previous-releases">control access to previous releases</a>.
+      </p>
+      <ReleaseContributorPermissions releaseId={newestRelease.id} />
+
+      {sortedReleases.length >= 0 && (
```

Comment:
This'll always be true, so no need for the check, or change the check to be `>` and only display `Previous releases` if not empty.